sdk/identity/Azure.Identity.Broker/src/Azure.Identity.Broker.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Azure.Identity" VersionOverride="1.14.0-beta.3" />
-    <PackageReference Include="Microsoft.Identity.Client" />
+    <PackageReference Include="Microsoft.Identity.Client" VersionOverride="4.71.1-preview" />
```

Comment:
Was seeing:

> sdk/identity/Azure.Identity.Broker/src/Azure.Identity.Broker.csproj#L0
sdk/identity/Azure.Identity.Broker/src/Azure.Identity.Broker.csproj(0,0): Error NU1605: Warning As Error: Detected package downgrade: Microsoft.Identity.Client from 4.71.1-preview to 4.71.0. Reference the package directly from the project to select a different version. 
 Azure.Identity.Broker -> Azure.Identity -> Microsoft.Identity.Client (>= 4.71.1-preview) 
 Azure.Identity.Broker -> Microsoft.Identity.Client (>= 4.71.0)